library/src/main/scala/scala/scalajs/js/Math.scala

```diff
@@ -278,4 +278,84 @@ object Math extends Object {
    * MDN
    */
   def tan(x: Double): Double = native
+
+  /**
+    * <span class="badge badge-ecma6" style="float: right;">ECMAScript 6</span>
+    * The Math.log1p() function returns the natural logarithm (base e) of 1 + a number
+    *
+    * @return The natural logarithm (base e) of 1 plus the given number. If the number
+    *         is less than -1, NaN is returned.
+    * MDN
+    */
```

Comment:
It is not consistent with the rest of the existing functions in the file, but will change it.